lasy/profiles/combined_profile.py

```diff
@@ -54,16 +54,16 @@ def __init__(self, wavelength, pol, laser_energy, long_profile, trans_profile):
 
     def evaluate(self, x, y, t):
         """
-        Returns the envelope field of the laser
+        Return the envelope field of the laser.
 
         Parameters
         ----------
         x, y, t: ndarrays of floats
             Define points on which to evaluate the envelope
             These arrays need to all have the same shape.
 
-        Returns
-        -------
+        Return
+        ------
```

Comment:
I think this should not be changed. According to the numpy style, this should be `Returns` (https://numpydoc.readthedocs.io/en/latest/format.html#returns)

Reply:
Oh, thanks!!